corehq/apps/userreports/tests/test_view.py

```diff
@@ -207,3 +211,51 @@ def test_redirect_report_builder(self):
         report.save()
         request = HttpRequest()
         self.assertTrue(view.should_redirect_to_paywall(request))
+
+    def test_can_edit_report(self):
+        """
+        Test whether ConfigurableReportView.page_context allows report editing
+        """
+        domain = Domain(name='test_domain', is_active=True)
+        domain.save()
```

Comment:
Can you also make sure the test cleans up these objects?  `self.addCleanup(domain.delete)` is probably the easiest way to do it.  That should also cascade delete the roles, but the web users will need to be deleted individually.

Reply:
Sure, I've added those calls to addCleanup.